src/main/jflex/flowg.flex

```diff
@@ -101,6 +105,11 @@ Anything = .
 "if" { return symbol("if", sym.IF); }
 "else" { return symbol("else", sym.ELSE); }
 
+// GCode
```

Comment:
All token rules (idk what to call it) that is not in a state should be in the <YYINITIAL> state